db_management/db_upgrade.rst

```diff
@@ -0,0 +1,422 @@
+
+Process workflow
+~~~~~~~~~~~~~~~~
+
+The upgrade process in a nutshell:
+
+#. You create a test upgrade request.
+#. | The request is processed by Odoo:
+   | This happens via an automated process that runs the database through an upgrade script and
+     takes between 20 and 120 minutes. Only if an issue(s) arises will we have to intervene
+     manually and adjust the script specifically to your database until the upgrade succeeds.
+#. Odoo delivers a test database.
+#. You test your database for possible discrepancies (see :ref:`db-upgrade/test-guidance`)
+#. If there are any discrepancies, you report them to the Upgrade support team via the Help portal.
+#. We will fix the issues and send you a new test database.
+#. Once you completed the testing and are happy with the result, you decide on a date and time when
+   you stop users from accessing Odoo, freeze all data entries and create an upgrade request for the
+   production upgrade.
+#. Odoo delivers the production database through the automated process.
+#. You restore it in your Production environment a few short hours later and continue working on the
+   newly upgraded database.
+
+.. _db-upgrade/service-level:
+
+Service Level Agreement
+-----------------------
+
+What is covered by the Enterprise Licence?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Databases hosted on Odoo’s Cloud platforms (Saas and Odoo.sh) or On-Premise (Self-Hosting) enjoy the
+following service at all times.
+
+The upgrade of:
+
+* standard applications
+* Studio customization (as long as the Studio app is still active)
+* customizations done by our consulting and developer services *if* they are covered by a
+  ‘Maintenance of Customisations’ subscription
+
+The Upgrade Service is limited to the technical conversion and adaptation of your database (standard
+modules and data) to make it compatible with the targeted Version.
+
+What is NOT covered?
+~~~~~~~~~~~~~~~~~~~~
+
+* The cleaning of pre-existing data & configuration while upgrading
+* Any new developments and/or upgrade of your own :ref:`custom modules <db-upgrade/custom-modules>`
+* `Training <https://www.odoo.com/learn>`_ on the new version
+
+You can get more information about your Enterprise Licence on our :ref:`Odoo Enterprise Subscription
+Agreement <upgrade>` page.
+
+.. note::
+   If you need Odoo assistance on this matter, please contact your Odoo Account Manager or contact
+   our `Sales department <mailto:sales@odoo.com>`_.
+
+.. _db-upgrade/get-started:
+
+Get started
+===========
+
+The upgrade process varies depending on where your database is hosted.
+
+.. _db-upgrade/online:
+
+Online (SaaS)
+-------------
+
+If you are hosted Online, please check our :ref:`instruction page for our Online (SaaS) customers
+<upgrade_button>`.
+
+.. _db-upgrade/odoo-sh:
+
+Odoo.sh
+-------
+
+If you are Odoo.sh hosted, check our :doc:`specific instructions to be able to upgrade
+<../odoo_sh/advanced/upgrade_your_database>`.
```

Comment:
It might be worth doing the same in those docs and refers to this one if customers are on premise.

Reply:
good idea, but if we could already push this documentation in prod and improve it afterward, it would be great

Reply:
That's really not a big piece of work. If you could tell @jcs-odoo in which doc pages (I assume the same that you referenced) those references should go, this would be a matter of minutes to have them in this PR too.

Reply:
@avg-odoo 
if you send me the docs and where to add the link by the end of the day, it can be added to this PR today indeed.
In the meanwhile, I already do and push the other changes :)